packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -242,7 +242,16 @@ class FlutterPlugin implements Plugin<Project> {
 
     private Properties getPluginList(Project project) {
         File pluginsFile = new File(project.projectDir.parentFile.parentFile, '.flutter-plugins')
-        return readPropertiesIfExist(pluginsFile)
+        Properties allPlugins = readPropertiesIfExist(pluginsFile)
+        Properties androidPlugins = new Properties()
+        allPlugins.each { name, path ->
+            File editableAndroidProject = new File(path, 'android' + File.pathSeparator + 'build.gradle')
```

Comment:
nit:  `File.separator` instead of `File.pathSeparator`. This should fix the tests that are failing.

Reply:
done, thanks.